invenio/modules/annotations/api.py

```diff
@@ -52,8 +54,8 @@ class Annotation(SmartJsonLD):
 
     @classmethod
     def create(cls, data, model='annotation'):
-        dic = Reader.translate(data, cls, model=model, master_format='json',
-                               namespace="annotationsext")
+        dic = translate(data, cls, metadata=MetaData(packages=[model]),
```

Comment:
NOTE `dic = translate(data, cls, metadata=get_metadata("annotationsext"),` (namespace is not used anymore)